store/src/elements.rs

```diff
@@ -376,7 +376,7 @@ where
 
       let cost = data["cost"]["number"].number_or_none();
 
-      if cost.is_none() {
+      if (qty.is_empty() || qty.is_zero()) && cost.is_none() {
```

Comment:
what is the difference between `is_empty` & `is_zero`? why `is_empty` at all?

Reply:
place both checks into is_zero() method